css/style.css

```diff
@@ -0,0 +1,44 @@
+body {
+  margin: 0 auto;
+}
+
+.navbar {
+  width: 1080px;
+  height: 150px;
+  border: 5px dotted blue;
+  float: left;
+  clear: right;
+}
+#container {
+  border: 5px solid red;
+  width: 1080px;
+  height: 520px;
+  clear: left;
+}
+
+#big {
+  width: 1080px;
+  height: 300px;
+  border: 5px solid black;
+  clear: left;
+}
+
+.flow-left {
+  border: 5px solid green;
+  width: 340px;
+  height: 500px;
+  float: left;
+  display: inline-block;
+  padding: 5px
+}
+
+.flow-left img {
+  text-align: center;
+  margin: auto;
+  width: 300px;
```

Comment:
You already give a width to the container element.  use the width + padding of that element to control the image size.  within the image, assign a width/ max-width of 100% to bound the image to it's container.